cmd/osv-scanner/main.go

```diff
@@ -34,6 +34,12 @@ func run(args []string, stdout, stderr io.Writer) int {
 		Writer:    stdout,
 		ErrWriter: stderr,
 		Flags: []cli.Flag{
+			&cli.StringSliceFlag{
```

Comment:
I think this should be a boolean flag rather than a slice, since my understanding is that there is only one expected `installed` file located at `/lib/apk/db/installed` - while some people might want to check arbitrary files as `installed`, this should be possible once #94 is landed and changing this flag afterwards would be a breaking change (+ worse-case if #94 stalls, we should easily add a `--apk-installed-files` flag alongside this one)